skops/card/_model_card.py

```diff
@@ -380,6 +373,8 @@ def _generate_card(self) -> ModelCard:
         # add evaluation results
 
         template_sections = copy.deepcopy(self._template_sections)
+        self._hyperparameter_table = self._extract_estimator_config()
```

Comment:
Do we even need to assign `self._hyperparameter_table` anymore? We could just do:

`hyperparameter_table = self._extract_estimator_config()`

and below, inside `ModelCard.from_template`, replace `hyperparameter_table=self._hyperparameter_table,` with `hyperparameter_table=hyperparameter_table,`.

Same argument goes for `self._model_plot`.

Reply:
Addressed this.